src/test/java/uk/gov/hmcts/cmc/claimstore/documents/content/DefendantResponseContentProviderTest.java

```diff
@@ -117,4 +117,14 @@ public void shouldProvideResponseTimeline() {
         assertThat(content)
             .containsKeys("timelineComment", "events");
     }
+
+    @Test
+    public void shouldProvideMediation() {
+        Map<String, Object> content = provider.createContent(claim);
+
+        assertThat(content)
+            .containsKey("freeMediation")
+            .containsValue("yes")
```

Comment:
This 'yes' is not necessarily the `freeMediation`'s value. Consider 
`assertThat(map).containsEntry(key, value)`
 instead

Reply:
done